docs/src/pages/guides/probes.md

```diff
@@ -89,6 +89,24 @@ probes:
         ping: true
 ```
 
+### Postgres Request
+
+Monika provides a way to check your postgres database's health with a 'postgres' type probe. You can define the `host`, `port` and `user` and `password` like this:
+
+```yaml
+probes:
```

Comment:
The common way to connect to a Postgres database is by using connection URI. It would be better if we can accept it as well. More info: https://www.prisma.io/dataguide/postgresql/short-guides/connection-uris

Reply:
+1 for this. Maybe if the `uri` is not empty, the user won't have to fill out `host`, `port`, `user,` and `password`. Vice versa

Reply:
Strangely enough `URI` connection string is not supported natively by the node-postgres library. So the URI string is now parsed into `host`, `port` etc.. and passed to the constructor.